dev-java/asm/asm-9.4.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+JAVA_PKG_IUSE="doc source test"
+MAVEN_PROVIDES="org.ow2.asm:asm-bom:9.4 org.ow2.asm:asm:9.4"
+# No tests because:
+# package org.junit.jupiter.api does not exist
+# package org.junit.jupiter.params does not exist
+# JAVA_TESTING_FRAMEWORKS="junit-4"
```

Comment:
it has this comment about no tests but tests seem to be implemented.

Reply:
> it has this comment about no tests but tests seem to be implemented.

removed test phase. thanks for reminding me.